### PR TITLE
Update approval-admin.md with e-Signature integration

### PR DESCRIPTION
**Pull request by ConnorR777**
Adding new fields to the Audit section as well as a brand new section around the e-Sign integration within Approvals.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/approval-admin.md [hunk 1]
```diff
@@ -148,4 +154,20 @@ Learn more about [Microsoft Dataverse and model-driven apps activity logging - P
 From the Teams Approvals app, users have access to create new Approvals and view Approvals that they have sent and received. Users won't have access to Approvals that are created by others unless they're either a responder or a viewer of the request.
 
 > [!Note]
-> A user will be given a viewer role of a request if they are part of the chat or channel where the approval was created. They won't have the ability to take action on the request if they weren't given that role when the approval was created.
\ No newline at end of file
+> A user will be given a viewer role of a request if they are part of the chat or channel where the approval was created. They won't have the ability to take action on the request if they weren't given that role when the approval was created.
+
+## Approvals e-Signature Integration
```
Comment (JohanFreelancer9) [inline Teams/approval-admin.md:159]:
```suggestion
## Approvals e-Signature integration
```

File: Teams/approval-admin.md [hunk 2]
```diff
@@ -148,4 +154,20 @@ Learn more about [Microsoft Dataverse and model-driven apps activity logging - P
...
+
+E-signature approvals created from the Approvals App are stored in the selected providers cloud environment. For further information regarding storage around the e-signature agreement please view their documentation around storage.
```
Comment (JohanFreelancer9) [inline Teams/approval-admin.md:161]:
```suggestion
e-Signature approvals created from the Approvals App are stored in the selected provider's cloud environment. For further information regarding storage around the e-Signature agreement, please view their documentation around storage.
```

File: Teams/approval-admin.md [hunk 3]
```diff
@@ -148,4 +154,20 @@ Learn more about [Microsoft Dataverse and model-driven apps activity logging - P
...
+
+To use the Approvals app e-Signature feature, you need the following items:
```
Comment (JohanFreelancer9) [inline Teams/approval-admin.md:163]:
```suggestion
To use the Approvals app e-Signature feature, you need a license for the specific e-Signature provider you are choosing to use. In order to obtain a license for your organization, you will need to go to the provider’s site.```

File: Teams/approval-admin.md [hunk 4]
```diff
@@ -148,4 +154,20 @@ Learn more about [Microsoft Dataverse and model-driven apps activity logging - P
...
+
```
Comment (JohanFreelancer9) [inline Teams/approval-admin.md:164]:
```suggestion
```

File: Teams/approval-admin.md [hunk 5]
```diff
@@ -148,4 +154,20 @@ Learn more about [Microsoft Dataverse and model-driven apps activity logging - P
...
+- License for the specific e-Signature provider you are choosing to use. In order to obtain a license for your organization you will need to go to the provider’s site.
```
Comment (JohanFreelancer9) [inline Teams/approval-admin.md:165]:
```suggestion
```

File: Teams/approval-admin.md [hunk 6]
```diff
@@ -148,4 +154,20 @@ Learn more about [Microsoft Dataverse and model-driven apps activity logging - P
...
+
+For the Approvals e-Signature functionality, Third party signature partners will appear in the Teams Approvals app by default. You can disable specific e-Signature providers through the app settings within the Teams admin center.
```
Comment (JohanFreelancer9) [inline Teams/approval-admin.md:167]:
```suggestion
For the Approvals e-Signature functionality, third-party signature partners will appear in the Teams Approvals app by default. You can disable specific e-Signature providers through the app settings within the Teams admin center.
```

File: Teams/approval-admin.md [hunk 7]
```diff
@@ -148,4 +154,20 @@ Learn more about [Microsoft Dataverse and model-driven apps activity logging - P
...
+
+1. Within the Approvals app under the Manage apps section in the Teams admin center select **Settings**
```
Comment (JohanFreelancer9) [inline Teams/approval-admin.md:169]:
```suggestion
1. Within the Approvals app under the Manage apps section in the Teams admin center, select **Settings**.
```

File: Teams/approval-admin.md [hunk 8]
```diff
@@ -148,4 +154,20 @@ Learn more about [Microsoft Dataverse and model-driven apps activity logging - P
...
+
+2.	Select the toggle for the specific eSignature provider to disable it. Should a Teams admin disable a provider, end users will not see the provider when creating an approval or be able to view any e-Signture requests that were made with that provider.
```
Comment (JohanFreelancer9) [inline Teams/approval-admin.md:171]:
```suggestion
2.	Select the toggle for the specific e-Signature provider to disable it. Should a Teams admin disable a provider, other users will not see the provider when creating an approval or view any e-Signature requests made with that provider. 
```

File: Teams/approval-admin.md [hunk 9]
```diff
@@ -148,4 +154,20 @@ Learn more about [Microsoft Dataverse and model-driven apps activity logging - P
...
+
+- License for the specific e-Signature provider you are choosing to use. In order to obtain a license for your organization you will need to go to the provider’s site.
+
+For the Approvals e-Signature functionality, Third party signature partners will appear in the Teams Approvals app by default. You can disable specific e-Signature providers through the app settings within the Teams admin center.
+
+1. Within the Approvals app under the Manage apps section in the Teams admin center select **Settings**
+
+2.	Select the toggle for the specific eSignature provider to disable it. Should a Teams admin disable a provider, end users will not see the provider when creating an approval or be able to view any e-Signture requests that were made with that provider. 
+
+E-signature Approvals created from the Approvals App are stored in the selected provider’s cloud. To export any data regarding eSign you will need to go to the providers site in order to do this. Please refer to their documentation about export and retention of these agreements.
```
Comment (JohanFreelancer9) [inline Teams/approval-admin.md:173]:
```suggestion
e-Signature Approvals created from the Approvals app are stored in the selected provider’s cloud. To export any data regarding e-Signature, you will need to go to the provider's site in order to do so. Please refer to their documentation about the export and retention of these agreements.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by ConnorR777** [comment]
@scanum All suggested changes were taken. Any other changes needed for us to approver this? The feature is currently in public preview and our customers are asking for this. Thanks!

File: Teams/approval-admin.md [hunk 10]
```diff
@@ -148,4 +154,18 @@ Learn more about [Microsoft Dataverse and model-driven apps activity logging - P
...
 > [!Note]
-> A user will be given a viewer role of a request if they are part of the chat or channel where the approval was created. They won't have the ability to take action on the request if they weren't given that role when the approval was created.
\ No newline at end of file
+> A user will be given a viewer role of a request if they are part of the chat or channel where the approval was created. They won't have the ability to take action on the request if they weren't given that role when the approval was created.
+
+## Approvals e-Signature integration
+
+e-Signature approvals created from the Approvals App are stored in the selected provider's cloud environment. For further information regarding storage around the e-Signature agreement, please view their documentation around storage.
+
+To use the Approvals app e-Signature feature, you need a license for the specific e-Signature provider you are choosing to use. In order to obtain a license for your organization, you will need to go to the provider’s site.```
```
Comment (JohanFreelancer9) [inline Teams/approval-admin.md:163]:
```suggestion
To use the Approvals app e-Signature feature, you need a license for the specific e-Signature provider you are choosing to use. In order to obtain a license for your organization, you will need to go to the provider’s site.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@ConnorR777 - Thank you for committing my suggestions. My apologies, I don't know how this happened, but could you please just remove the extra characters that somehow ended up in my suggestion. Many thanks.

**Review by scanum — APPROVED** [review]

**Pull request closed without merging**